share/man/man4/uart.4

```diff
@@ -23,12 +25,12 @@
 .\" (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 .\" THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 .\"
-.Dd October 11, 2024
+.Dd November 17, 2024
 .Dt UART 4
 .Os
 .Sh NAME
 .Nm uart
-.Nd driver for Universal Asynchronous Receiver/Transmitter (UART) devices
+.Nd Universal Asynchronous Receiver/Transmitter serial driver
```

Comment:
UART really needs to be in the Nd.

Reply:
Even though it's in the Nm? Afaik, they're always rendered together.